lib/mappers/index.js

```diff
@@ -1,13 +1,30 @@
-const mapTextStyle = require('./textstyle');
-const mapColor = require('./color');
-const mapGradient = require('./gradient');
-const mapShadow = require('./shadow');
-const mapBorder = require('./border');
-
-module.exports = {
-  mapColors: colors => colors.map(mapColor),
-  mapTextStyles: textStyles => textStyles.objects.map(mapTextStyle),
-  mapGradients: gradients => gradients.map(mapGradient),
-  mapShadows: shadows => shadows.map(mapShadow),
-  mapBorders: borders => borders.map(mapBorder),
+const mapTextStyleSketch = require('./sketch/textstyle');
+const mapColorSketch = require('./sketch/color');
+const mapGradientSketch = require('./sketch/gradient');
+const mapShadowSketch = require('./sketch/shadow');
+const mapBorderSketch = require('./sketch/border');
+
+const mapTextStyleFigma = require('./figma/textstyle');
+const mapColorFigma = require('./figma/color');
+const mapGradientFigma = require('./figma/gradient');
+const mapShadowFigma = require('./figma/shadow');
+const mapBorderFigma = require('./figma/border');
```

Comment:
Place a module mapper at `sketch/index.js` and `figma/index.js` and you can do this:

```js
// figma/index.js
module.exports = {
  colors: mapColorFn,
  ...
};

// this file
const mapFigma = require('./figma');

sketch ? mapSketch.colors : mapFigma.colors;

```